src/ResourceManager/Network/Commands.Network.Test/Commands.Network.Test.csproj

```diff
@@ -94,6 +102,7 @@
     <Compile Include="ScenarioTests\NetworkWatcherAPITests.cs" />
   </ItemGroup>
   <ItemGroup>
+    <None Include="app.config" />
```

Comment:
why do we need app.config?

Reply:
it must have been updated by auto-generator. Will remove it